### PR TITLE
Fixes #3734:  Improve google Lighthouse accessibly score #3734

### LINKED ISSUE #3734: Improve google Lighthouse accessibly score
Google Lighthouse accessibly score is 97%
Screen Shot 2022-10-23 at 8 21 58 PM

**What would you like to be added**:
Improve accessibly score to 100%
**Why would you like this to be added**:
Improve accessibility for visually impaired people

### PR DESCRIPTION
**Pull request by IvaniGabrovsky**
## Issue This PR Addresses

Fixes #3734

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Improve accessibly score to 100% by making colour darker shade of grey to be visible for visually impaired people.
## Steps to test the PR
    - Go to 'app' directory
    - `pnpm install`
    - In src/components/Posts/index.tsx change the colour of error and error icon
    - Open url and go to 'http://localhost:8000/'
    - Scroll down to 'Blog Timeline Failed to Load!'

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/app/src/components/Posts/index.tsx [hunk 1]
```diff
@@ -21,12 +21,12 @@ const useStyles = makeStyles((theme: Theme) =>
       width: '100%',
     },
     error: {
-      color: '#B5B5B5',
+      color: '#4d4d4d',
```
Comment (manekenpix) [inline src/web/app/src/components/Posts/index.tsx:24]: Can we try to add this our themes so we don't have to hardcode colours? Same for `errorIcon`.

File: src/web/docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ const config = async () => {
     ],
     i18n: {
       defaultLocale: 'en',
-      locales: ['en', 'vi'],
+      locales: ['en', 'vi', 'bg'],
```
Comment (manekenpix) [inline src/web/docusaurus/docusaurus.config.js:44]: I think this is part of a different PR, along with the changes in `overview.md`.

**Review by manekenpix — CHANGES_REQUESTED** [review]
This PR needs to remove the changes from another PR and sync its branch with upstream master.

File: src/web/docusaurus/docusaurus.config.js (hunk 2 above)
Comment (IvaniGabrovsky) [reply src/web/docusaurus/docusaurus.config.js]: Yes this is from the wrong PR. I have removed the changes.

File: src/web/app/src/components/Posts/index.tsx (hunk 1 above)
Comment (IvaniGabrovsky) [reply src/web/app/src/components/Posts/index.tsx]: I am using colour from themes.

**Review by manekenpix — CHANGES_REQUESTED** [review]
@IvaniGabrovsky can you please fix the history in your branch? Rebasing is not supposed to add new commits to a branch, and you have one commit for that. Also, you merged master into your branch a few times, which is not necessary when you properly rebase your branch on `upstream/master`.

**Comment by humphd** [comment]
@IvaniGabrovsky this one needs squashing/rebasing too.  If you're hoping to use for Hacktoberfest, we need it done ASAP, as today is the last day.

**Review by manekenpix — APPROVED** [review]
LGTM :+1: 

image

**Pull request merged**